ui/store/actions.js

```diff
@@ -51,9 +52,37 @@ import * as actionConstants from './actionConstants';
 
 let background = null;
 let promisifiedBackground = null;
+const retryQueue = [];
```

Comment:
1. Since the file is quite large, I wonder if it would help if we renamed `retryQueue` to be more specific?

---- 
2. Would we want this queue to be persisted should anything happen to the user's session? Should we consider storing the data e.g. chrome.storage?

Reply:
Yeah please can you help me do that to work perfectly please